lib/fastlane/plugin/create_remote_config_release_package/actions/create_remote_config_release_package_action.rb

```diff
@@ -163,6 +169,16 @@ def self.available_options
             verify_block: proc do |value|
               UI.user_error!("No Xcode project path given, pass using `project_file_path: 'project_file_path'`") unless value && !value.empty?
             end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :target_name,
+            env_name: "RC_TARGET",
+            description: "specifies the target",
+            optional: true,
+            type: String,
+            verify_block: proc do |value|
+              UI.user_error!("No target given, pass using `target_name: 'target_name'`") unless value && !value.empty?
+            end
```

Comment:
We can remove this `verify_block` so that we'll not block the user if this value is not provided. [More Info](https://docs.fastlane.tools/advanced/actions/)

Otherwise, all looks 🔥